packages/terra-dropdown-button/src/_DropdownList.jsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
```

Comment:
I assume we investigated using terra-menu but couldn't?

Reply:
I hadn't actually, although it looks to be different in keyboard interaction (menu uses tab instead of up/down arrow), some style variances, it appears to prefer to open left instead of right, and uses `Popup` which we moved away from to avoid issues with mobile screens due to not being able to scroll away from the dropdown.